packages/lab/stories/combo-box-next/combo-box-next.doc.stories.mdx

```diff
@@ -58,6 +58,23 @@ Tip: For recommendations on variant choice, see the <Link href="https://www.salt
   <Story id="lab-combo-box-next--secondary" />
 </Canvas>
 
+### Placeholder
+
+You can use a placeholder to prompt user input if no default value is provided.
+
+<Canvas>
+  <Story id="lab-combo-box-next--placeholder" />
+</Canvas>
+
+### Controlled
+
+You can control the `value` and `open` and `selected` props of Combo Box.
+Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components</Link> in the official React docs.
```

Comment:
in uitk we have in the doc: 
"When used as a controlled ... how Combo Box interacts with user input. In this example, the selected text is transformed to upper case."

I think we could put here what/how it's strictly controlled by the buttons, to be more clear on the controlled implementation
I wonder if we should be similar with our controlled stories, e.g. this/dropdown using buttons whereas Input controlled story isnt really different from the default, could probably be improved

Reply:
will follow up with this update